golangServer/server.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"Oracle.com/golangServer/Oracle"
+	"Oracle.com/golangServer/api"
+	"Oracle.com/golangServer/config"
+	"Oracle.com/golangServer/model"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func init() {
```

Comment:
We should also get the data storage path passed in with something like `~/.config/prolly-tree-oracle/` as the default.